utils/nerdctl/Makefile

```diff
@@ -0,0 +1,47 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=nerdctl
+PKG_VERSION:=1.6.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/containerd/nerdctl/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=a369b1c517d9c3d53d00b29633a6176a05811214a44dd25d339c32cc6a901579
+
+PKG_MAINTAINER:=Luca Barbato <lu_zero@gentoo.org>
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+PKG_BUILD_FLAGS:=no-mips16
+
+GO_PKG:=github.com/containerd/nerdctl
+
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/golang/golang-package.mk
+
+define Package/nerdctl
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=contaiNERD CTL - Docker-compatible CLI for containerd
+  URL:=https://containerd.io/
+  DEPENDS:=$(GO_ARCH_DEPENDS)
+endef
+
+define Package/nerdctl/description
+  Docker-compatible CLI for containerd, with support for Compose, Rootless, eStargz, OCIcrypt, IPFS, ...
+endef
+
+GO_PKG_BUILD_VARS += GO111MODULE=auto
```

Comment:
I haven't look at it closely, but why do we need to have it? We do have Go 1.12 in the master and openwrt-23.05 branches.

Reply:
I copied what's in the docker makefile, I wanted to see if nerdctl + cni-plugins would be lighter than dockerd+docker+docker-compose

Reply:
Okay, so we just copied something from somewhere and we don't know even why and what it does. Am I wrong?

Reply:
My faulty assumption is that the docker Makefile could be copied as-is and I forgot to make this as draft. If you have a good Makefile to point me to I'd be grateful.

Reply:
https://github.com/openwrt/packages/commit/0187a8d55ab22e4a77659659b2032586c4bbf894#r129204807

Reply:
I removed all the noise inherited from the docker Makefile and it seems still working fine from my limited testing.